content/en/flux/installation/_index.md

```diff
@@ -0,0 +1,308 @@
+but we don't recommend running [EOL versions](https://endoflife.date/kubernetes)
+in production nor do we offer support for these versions.
+{{% /alert %}}
+
+## Install the Flux CLI
+
+The Flux CLI is available as a binary executable for all major platforms,
+the binaries can be downloaded from GitHub
+[releases page](https://github.com/fluxcd/flux2/releases).
+
+{{< tabpane text=true >}}
+{{% tab header="Homebrew" %}}
+
+With [Homebrew](https://brew.sh) for macOS and Linux:
+
+```sh
+brew install fluxcd/tap/flux
+```
+
+{{% /tab %}}
+{{% tab header="bash" %}}
+
+With [Bash](https://www.gnu.org/software/bash/) for macOS and Linux:
+
+```sh
+curl -s https://fluxcd.io/install.sh | sudo bash
+```
+
+{{% /tab %}}
+{{% tab header="yay" %}}
+
+With [yay](https://github.com/Jguer/yay) (or another [AUR helper](https://wiki.archlinux.org/title/AUR_helpers)) for Arch Linux:
+
+```sh
+yay -S flux-bin
+```
+
+{{% /tab %}}
+{{% tab header="nix" %}}
+
+With [nix-env](https://nixos.org/manual/nix/unstable/command-ref/nix-env.html) for NixOS:
+
+```sh
+nix-env -i fluxcd
+```
+
+{{% /tab %}}
+{{% tab header="Chocolatey" %}}
+
+With [Chocolatey](https://chocolatey.org/) for Windows:
+
+```powershell
+choco install flux
+```
+
+{{% /tab %}}
+{{< /tabpane >}}
+
+To configure your shell to load `flux` [bash completions](./cmd/flux_completion_bash.md) add to your profile:
+
+```sh
+. <(flux completion bash)
+```
+
+[`zsh`](./cmd/flux_completion_zsh.md), [`fish`](./cmd/flux_completion_fish.md),
+and [`powershell`](./cmd/flux_completion_powershell.md)
+are also supported with their own sub-commands.
+
+A container image with `kubectl` and `flux` is available on DockerHub and GitHub:
+
+* `docker.io/fluxcd/flux-cli:<version>`
+* `ghcr.io/fluxcd/flux-cli:<version>`
+
+## Bootstrap
+
+Using the `flux bootstrap` command you can install Flux on a
+Kubernetes cluster and configure it to manage itself from a Git
+repository.
+
+If the Flux components are present on the cluster, the bootstrap
+command will perform an upgrade if needed. The bootstrap is
+idempotent, it's safe to run the command as many times as you want.
+
+The Flux component images are published to DockerHub and GitHub Container Registry
+as [multi-arch container images](https://docs.docker.com/docker-for-mac/multi-arch/)
+with support for Linux `amd64`, `arm64` and `armv7` (e.g. 32bit Raspberry Pi)
+architectures.
+
+If your Git provider is **AWS CodeCommit**, **Azure DevOps**, **Bitbucket Server**, **GitHub** or **GitLab** please
+follow the specific bootstrap procedure:
+
+* [AWS CodeCommit](./bootstrap/aws-code-commit.md#flux-installation-for-aws-codecommit)
+* [Azure DevOps](./bootstrap/azure.md#flux-installation-for-azure-devops)
+* [Bitbucket Server and Data Center](./bootstrap/bitbucket.md#bitbucket-server-and-data-center)
+* [GitHub.com and GitHub Enterprise](./bootstrap/github.md#github-and-github-enterprise)
+* [GitLab.com and GitLab Enterprise](./bootstrap/gitlab.md#gitlab-and-gitlab-enterprise)
```

Comment:
We need to list here all the options with Generic Git server first, also no need to repeat the provider names in bold.